### ISSUE TITLE
TypeError: vnode.createInstance is not a function when calling vnode.createInstance(props);

### ISSUE THREAD

**Issue by wachunei** [maticzav/emma-cli#13]
Fresh install, running emma

```sh
~/.config/yarn/global/node_modules/emma-cli/node_modules/ink/lib/diff.js:32
                vnode.createInstance(props);
                      ^

TypeError: vnode.createInstance is not a function
```

**Comment by wachunei** [comment]
Hello 👋! 

I'm sorry for the poor details of the issue, should have provided more info.

I'm using node v8.11.1 (Latest LTS: Carbon).

**Comment by maticzav** [comment]
Could you provide a more detailed explanation of the issue? I tried to reproduce it but wasn't successful in doing so.

**Comment by wachunei** [comment]
what I did:

```
yarn global add emma-cli
emma
```

<img width="1284" alt="screen shot 2018-05-18 at 6 16 38 pm" src="https://user-images.githubusercontent.com/1024246/40260279-bb9476fa-5ac7-11e8-8042-8452762503c4.png">

**Comment by vadistic** [comment]
Hi, I have the same issue :<

`Windows 10 (May update), yarn 1.7, node 10.0.0`

Exactly the same error & callstack. 

Steps:
1. Clean yarn install to update it to 1.7
2. `yarn global add emma-cli` and `emma`

Some things I tried:
1) Quickfix: It works if I `git clone https://github.com/maticzav/emma-cli/`  & `yarn link`
2) It still does not work if I then build it and substitute emma-cli dist in node_modules (same error)
3) I tired updating deps (it looka like typeerror with `ink` and there is new `ink^0.5.0`, `@babel/*` versions are out of sync with each other and `child_process` dep looks depreciated), but no luck :)
4) I tried running it on Bash/Ubuntu via Windows Subsystem for Linux and same error
